Flow.Launcher.Core/Configuration/Portable.cs

```diff
@@ -142,7 +143,8 @@ public void CreateUninstallerEntry()
 
         internal void IndicateDeletion(string filePathTodelete)
         {
-            using (StreamWriter sw = File.CreateText(filePathTodelete + "\\" + DataLocation.DeletionIndicatorFile)){}
+            var deleteFilePath = Path.Combine(filePathTodelete, DataLocation.DeletionIndicatorFile);
+            File.CreateText(deleteFilePath).Close();
```

Comment:
why are we not using using(){} for this?

Reply:
again, matter of preference; this is equivalent to empty using statement but nicer, IMO...

Reply:
maybe we should wrap it in using please, just so we cater for some unforeseen exception down the  track

Reply:
`using` does not suppress exceptions, it just disposes the object in case of exception... and `IndicateDeletion` is used in two places in the same file, `DisablePortableMode` and `EnablePortableMode`, both of which do not handle exceptions (we should probably change this behavior).

But okay, I'll switch back to a `using` statement here.